internal/core/processing.go

```diff
@@ -125,6 +126,22 @@ func ProcessBookmark(req ProcessRequest) (book model.Bookmark, isFatalErr bool,
 		}
 	}
 
+	// If needed, create ebook as well
+	if book.CreateEbook {
+		ebookPath := fp.Join(req.DataDir, "ebook", fmt.Sprintf("%d.epub", book.ID))
```

Comment:
We will need to be cautious in the future and do not remove content, archive or bookmark if we find problems when updating the cache. We could save the new one to a temporary location while we work, and if everything succeeds, replace the current one. We can create an issue for this.

Reply:
Yeah sure, it was just a comment so we don't forget this in the future, and we can fix everything at once.